zkevm-circuits/src/util/word.rs

```diff
@@ -70,6 +77,56 @@ impl<F: Field, const N: usize> WordLimbs<Cell<F>, N> {
         })
     }
 
+    // N_LO, N_HI are number of bytes to assign to first half and second half of size N limbs,
+    // respectively N_LO and N_HI can be different size, the only requirement is N_LO % (N/2)
+    // and N_HI % (N/2) [N/2] limbs will be assigned separately.
+    // E.g. N_LO = 4 => [nl1, nl2, nl3, nl4]
+    // N_HI = 2 => [nh1, nh2]
+    // N = 2 => [l1, l2]
+    // it equivalent l1.assign(nl1.expr() + nl2.expr() * 256 + nl3.expr() * 256^2 +  nl3.expr() *
+    // 256^3) and l2.assign(nh1.expr() + nh2.expr() * 256)
+    pub fn assign_lo_hi<const N_LO: usize, const N_HI: usize>(
```

Comment:
I think it will be very convenient to have an assign function that just takes a `eth_types::Word`, because that's the most common type we find from the witness data for Words.
Then the assignment can take the Word, turn it into bytes, and group them as necessary to assign each limb.

Reply:
Nice suggestion 🙏 I added `assign_u256`